src/js/components/ToggleGroup/stories/Controlled.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { Box, ToggleGroup, Text } from 'grommet';
+
+export const ControlledGroup = () => {
+  const [multipleControlled, setMultipleControlled] = useState(['c2']);
+
+  return (
+    <Box direction="row" gap="xlarge" overflow="auto">
+      <Box gap="large" pad="large">
+        <Text>An option always has to be checked</Text>
```

Comment:
How is this different than the default/simple behavior? Shouldn't this story be updated to show only one or none option can be selected?

Reply:
I believe in the `multiple` case we are allowing removal of all selections.

However, for clarity, I'd be supportive of changing this to be single select + ability to remove all selections.

Reply:
After discussion, we'll leave as is and just adjust the intro text for clarity.